src/controllers/users_controller.ts

```diff
@@ -26,10 +45,30 @@ export const signupByKakao = async (req: Request, res: Response) => {
 };
 
 export const signin = async (req: Request, res: Response) => {
-  const {} = req.body;
+  const { email, password }: { email: string; password: string } = req.body;
+
+  if (!email || !password) {
+    return res.status(statusCode.BAD_REQUEST).json(util.successFalse(resMessage.NULL_VALUE));
+  }
 
   try {
-  } catch (err) {}
+    const user = await usersService.readOneByEmail(email);
+    if (user == null) {
```

Comment:
```suggestion
    if (!user) {
```
이게 더 깔끔하지 않을까.... 싶습니당